src/styles/variables.less

```diff
@@ -44,6 +44,10 @@
 @color-gray-25: fade(#000, 25%);
 @color-gray-30: fade(#000, 30%);
 
+// Shadows
+
+@shadow-inset: 0 10px 10px -10px @color-gray-30 inset;
```

Comment:
I was thinking of `shadow` as another top level `type`. Maybe it should be `shadow-boxShadowInset` or something? Just trying to stick to a convention we lay out in [contrib](https://github.com/appnexus/lucid/blob/master/CONTRIBUTING.md)